school_library_decorator/person.rb

```diff
@@ -25,6 +25,53 @@ def add_rental(date, person)
     Rental.new(self, date, person)
   end
 
+  def self.all_people(people)
+    if people.length.positive?
+      people.each do |person|
+        puts "name: #{person.name}, age: #{person.age}, id: #{person.id}"
+      end
+    else
+      puts 'Sorry! cannot find person'
+    end
+  end
```

Comment:
- [ ] Kindly refactor this piece of code to follow the Single responsibility rule, in ruby we can also do more by writing less